tests/store/test_hdfs_artifact_repo.py

```diff
@@ -44,7 +46,7 @@ def test_log_artifact_with_kerberos_setup(hdfs_system_mock):
     os.environ['MLFLOW_KERBEROS_USER'] = 'some_kerberos_user'
     os.environ['MLFLOW_HDFS_DRIVER'] = 'libhdfs3'
 
-    repo = HdfsArtifactRepository('hdfs:/some/maybe/path')
+    repo = HdfsArtifactRepository('hdfs:///some/maybe/path')
```

Comment:
why 3 slashes?

Reply:
The format is `{schema}://{host}/{path}` if `schema = hdfs` and `host = root` it becomes `hdfs://root/user` 
`root` can be omitted, it becomes `hdfs:///user`